tests/compile-fail/concurrency/libc_pthread_rwlock_write_read_deadlock.rs

```diff
@@ -0,0 +1,32 @@
+// ignore-windows: No libc on Windows
```

Comment:
We already have a "sync" folder with other pthread tests. Either merge the two folddlers, or at least move everything synchronization-related to `sync`.